src/content-scripts/content-script.ts

```diff
@@ -215,7 +215,7 @@ const injectInPageScript = (isPriority) => {
 
 getIsPriorityWallet()
   .then((isPriorityWallet) => {
-    injectInPageScript(isPriorityWallet);
+    requestAnimationFrame(() => injectInPageScript(isPriorityWallet));
```

Comment:
What's the purpose of this? 🤔

Reply:
adding the `document_start` flag was crashing this function with an error that `dcoument.body` is undefined, this gives it a little time to load and doesn't block the thread to add the provider to the page